### PR TITLE
On import validation avoid time check on activation token

### PR DESCRIPTION
**Pull request by matthiashanel**
Signed-off-by: Matthias Hanel <mh@synadia.com>

### CODE REVIEW

File: v2/activation_claims.go [hunk 1]
```diff
@@ -105,7 +105,14 @@ func (a *ActivationClaims) Payload() interface{} {
 
 // Validate checks the claims
 func (a *ActivationClaims) Validate(vr *ValidationResults) {
-	a.ClaimsData.Validate(vr)
+	a.validateExt(vr, true)
+}
+
+// Validate checks the claims
+func (a *ActivationClaims) validateExt(vr *ValidationResults, timeChecks bool) {
```
Comment (aricart) [inline v2/activation_claims.go:112]: validateWithTimeChecks?

**Review by aricart — APPROVED** [review]
LGTM - pending tests passing in travis

File: v2/activation_claims.go (hunk 1 above)
Comment (matthiashanel) [reply v2/activation_claims.go]: fixed

**Review by aricart — APPROVED** [review]
LGTM

**Pull request merged**